wifiphisher/common/accesspoint.py

```diff
@@ -174,32 +186,32 @@ def start(self):
             "ssid": self.essid,
             "interface": self.interface,
             "channel": self.channel,
-            "karma_enable": 1,
             "deny_macs": self.deny_mac_addrs,
-            "wpspbc": True
         }
         if self.psk:
             hostapd_config['wpa2password'] = self.psk
-
-        # create the option dictionary
-        hostapd_options = {
-            'mute': True,
-            "eloop_term_disable": True
-        }
-
-        try:
-            self.hostapd_object = apctrl.Hostapd()
-            self.hostapd_object.start(hostapd_config, hostapd_options)
-        except KeyboardInterrupt:
-            raise Exception
-        # when roguehostapd fail to start rollback to use the hostapd
-        # on the system
-        except BaseException:
-            hostapd_config = {}
-            hostapd_options = {}
+        self.hostapd_object = apctrl.Hostapd()
+        if not self.force_hostapd:
+            try:
+                hostapd_config["karma_enable"] = 1
+                hostapd_config["wpspbc"] = True
+                hostapd_options = {
+                    'mute': True,
+                    "eloop_term_disable": True
+                }
+                self.hostapd_object.start(hostapd_config, hostapd_options)
+            except KeyboardInterrupt:
+                raise Exception
+            except BaseException:
+                print("[" + constants.R + "!" + constants.W + "] " +
+                      "roguehostapd is not installed! You can supply --force-hostapd"
+                      " to use the hostapd installed on the system instead")
```

Comment:
exit program when roguehostapd fail to lunch but give users hint to use --force-hostapd to lunch the AP by hostapd

Reply:
I would change the message to:

"Roguehostapd is not installed in the system! Please install roguehostapd manually (https://github.com/wifiphisher/roguehostapd) and rerun the script. Otherwise, you can run the tool with the --force-hostapd option to use hostapd but please note that using Wifiphisher with hostapd will turn off many significant features of the tool."